user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Basic_controls/DashboardSearchInputFeed.md

```diff
@@ -0,0 +1,23 @@
+---
+uid: DashboardSearchInputFeed
+---
+
+# Search input
+
+Available from DataMiner 10.4.7 onwards<!--  RN 35902 -->. This basic control allows the user to enter text, which will then be available as a string feed in the dashboard or low-code app. The string feed can be used by queries and by script parameters in low-code app actions. This string feed can also be cleared with an 'x' button available in the input.
```

Comment:
I suspect the RN number mentioned here is incorrect, as 35902 is the RN introducing the text input in DataMiner 10.3.5. Could you add the correct number?